libs/src/constants.ts

```diff
@@ -0,0 +1,129 @@
+import type { OracleType } from "./oracle2/types";
+import { ethers } from "ethers";
+
+const getAddress = ethers.utils.getAddress;
+type ContractType = OracleType | "Erc20";
+type ContractInfo = {
+  chainId: number;
+  address: string;
+  type: ContractType;
+};
+type ContractInfoList = ContractInfo[];
+export const ContractInfoList: ContractInfoList = [
```

Comment:
i think theres a library for this, but hard coding it for now. unfortunately its hard for the graph to return the oracle addresses in the query, so we need to augment the data so that we know what spender to approve when doing token approval